lms/static/js/spec/financial-assistance/financial_assistance_form_view_spec.js

```diff
@@ -36,10 +94,48 @@ define([
                     dashboard_url: '/dashboard',
                     platform_name: 'edx',
                     submit_url: '/api/financial/v1/assistance'
-                };
+                },
+                completeForm = {},
```

Comment:
That's fair, I usually assign a default value to vars but in this case there is no point. I will update